### PR TITLE
EXLM-1012 Solutions field added for Browse Filters

### PR DESCRIPTION
**Pull request by ruhisingh1**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-1012

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/browse
- After: https://feature-exlm-1012-ue--exlm--adobe-experience-league.hlx.live/en/browse

https://experience.adobe.com/#/@amc/aem/editor/canvas/author-p122525-e1200861.adobeaemcloud.com/content/exlm/global/en/browse.html?ref=feature-EXLM-1012-UE

### CODE REVIEW

File: blocks/browse-filters/browse-filters.js [hunk 1]
```diff
@@ -757,9 +757,14 @@ function renderSortContainer(block) {
...
+  const thirdChild = block.querySelector('div:nth-child(3)');
   const headingElement = block.querySelector('div:nth-child(1) > div');
-  const topics = block.querySelector('div:nth-child(2) > div').textContent.trim();
-  const allTopicsTags = topics !== '' ? formattedTopicsTags(topics) : '';
+  const solutions = block.querySelector('div:nth-child(2) > div').textContent.trim();
+  const topics = block.querySelector('div:nth-child(3) > div').textContent.trim();
+  const allSolutionsTags = solutions !== '' ? formattedTags(solutions) : '';
+  const allTopicsTags = topics !== '' ? formattedTags(topics) : '';
+  // eslint-disable-next-line no-console
+  console.log(allSolutionsTags);
```
Comment (noormd08) [inline blocks/browse-filters/browse-filters.js:767]: Please remove the console and also can we cleanup the above code with map?

File: blocks/browse-filters/browse-topics.js [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+    const [type, productBase64, versionBase64] = item.split('/');
+    if (productBase64 !== undefined) {
+      product = atob(productBase64);
+    }
+    if (versionBase64 !== undefined) {
+      version = atob(versionBase64);
+    }
+
+    // Check if product and version are not undefined before appending to base64EncodedTagsArray
+    if (product !== undefined || version !== undefined) {
```
Comment (noormd08) [inline blocks/browse-filters/browse-topics.js:24]: There is possibility that we can get empty values as well. It is better to go with default empty/undefined/null checks.
ex: if (productBase64)

Comment (ruhisingh1) [reply blocks/browse-filters/browse-topics.js]: Done

File: blocks/browse-filters/browse-filters.js (hunk 1 above)
Comment (ruhisingh1) [reply blocks/browse-filters/browse-filters.js]: Done

**Review by nitin-rachabathuni — APPROVED** [review]

**Review by noormd08 — APPROVED** [review]

**Pull request merged**